G1ANT.Addon.IExplorer/G1ANT.Addon.IE.Tests/Commands/ie.switch/ShouldCloseAll_WhenSwitchMultiple.G1ANT

```diff
@@ -0,0 +1,24 @@
+﻿addon iexplorer version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+procedure OpenIE 
+    ie.open result  ♥IE_Handle1
+    ie.open result  ♥IE_Handle2
+end procedure
+
+procedure SwitchIE
+    ie.switch ♥IE_Handle1
+    ie.switch ♥IE_Handle2
+end procedure
+
+procedure SwitchAndCloseIE
+    ie.switch ♥IE_Handle1
+    ie.close
+    ie.switch ♥IE_Handle2
+    ie.close
+end procedure
+
+call OpenIE
+call SwitchIE
```

Comment:
What does it test? You do the switches, but don't verify the result of the action.

Reply:
a[href*='mail.google']
was added in: ShouldChangeTitle_WhenClickURL.G1ANT
its also used in ie.fireevent